compiler/rustc_typeck/src/check/pat.rs

```diff
@@ -1029,17 +1030,59 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
                 // last sub-pattern. In the case of `A(x)` the first and last may coincide.
                 // This looks like:
                 //
-                // help: missing parenthesis
+                // help: missing parentheses
                 //   |
                 // L |     let A((x, y)) = A((1, 2));
                 //   |           ^    ^
                 [first, ..] => (first.span.shrink_to_lo(), subpats.last().unwrap().span),
             };
             err.multipart_suggestion(
-                "missing parenthesis",
+                "missing parentheses",
```

Comment:
```suggestion
                "missing parenthesis",
```
Unnecessary change, it became a typo.

Reply:
No, it was a typo _before_.